golem-cli/src/oss/main.rs

```diff
@@ -126,6 +126,10 @@ pub async fn async_main<ProfileAdd: Into<UniversalProfileAdd> + clap::Args>(
 
             get_resource_by_uri(uri, &factory).await
         }
+        OssCommand::Completion { .. } => {
```

Comment:
Please add the same command to `init.rs`

Reply:
added